subsystem/kubeapi_test.go

```diff
@@ -2054,11 +2056,12 @@ var _ = Describe("[kube-api]cluster installation", func() {
 			return aci.Status.DebugInfo.EventsURL
 		}, "30s", "10s").ShouldNot(Equal(""))
 
-		By("Check ACI Logs URL exists")
+		By("Check ACI Logs URL is empty")
+		// Should not show the URL since no logs were collected
 		Eventually(func() string {
 			aci := getAgentClusterInstallCRD(ctx, kubeClient, installkey)
 			return aci.Status.DebugInfo.LogsURL
-		}, "30s", "10s").ShouldNot(Equal(""))
+		}, "30s", "10s").Should(Equal(""))
```

Comment:
We should have at least one check that the URL is populated.
There a some helper methods in sub system to simulate log collection: `updateClusterLogProgress` and `updateHostLogProgress`

Reply:
done.
I used the UploadLogs API calls, as this is the relevant API